### ISSUE TITLE
Events (as function) behaving not correct on the depending view 

### ISSUE THREAD

**Issue by kdenzel** [fullcalendar/fullcalendar-scheduler#445]
I know this issue was already confirmed from my other account but because of trolling it is now flagged, so i do it again with my real account.

See jsbin. Open the output window to see the description

https://jsbin.com/jipozedohu/1/edit?html,output

Problem is: If you load the year view, then switch back to the month view and manually reload the events for the monthview, the parameters 'start' and 'end'-momentObjects are still in the 'year'-mode and starts from YYYY-1-1 til YYYY-12-31 instead of 1st of month til end of month.

**Comment by acerix** [comment]
Thanks for the report.  Was the original issue deleted? I can't find it.

**Comment by kdenzel** [comment]
Yes that's the reason i reopened it. The account itself was not deleted but tagged in some way so other users can't see posts.
